tests/org.jboss.tools.aerogear.ui.bot.test/src/org/jboss/tools/aerogear/ui/bot/test/utils/ResourceUtils.java

```diff
@@ -0,0 +1,24 @@
+package org.jboss.tools.aerogear.ui.bot.test.utils;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.nio.MappedByteBuffer;
+import java.nio.channels.FileChannel;
+import java.nio.charset.Charset;
+
```

Comment:
This is a standard java library. I don't see anything bad in reading a file. Where do you want me to put the code? Calling RedDeer (can you point me where is the code?) is fine, but depending on some other 'common' plugin is a no go for me.